app/views/layouts/application.html.haml

```diff
@@ -5,6 +5,7 @@
     %meta{:name => 'viewport', :content => 'width=device-width, initial-scale=1'}/
     %meta{'http-equiv' => 'X-UA-Compatible', :content => 'IE=edge'}/
 
+    %link{:rel => "icon", :href => "#{favicon_path}", :type => "image/x-icon"}/
```

Comment:
The quotes and interpolation around the helper method are optional/redundant here.

```
%link{:rel => "icon", :href => favicon_path, :type => "image/x-icon"}/
```